editor.planx.uk/src/routes/flow.tsx

```diff
@@ -174,98 +176,83 @@ const nodeRoutes = mount({
-          flowSettings: settings
-          flowAnalyticsLink: analytics_link
-          publishedFlowsAggregate: published_flows_aggregate {
-            aggregate {
-              count
-            }
-          }
-        }
-      }
-    `,
-    variables: {
-      slug: flowSlug,
-      team_slug: team,
-    },
-  });
-
-  const flow = flows[0];
-  if (!flows) throw new NotFoundError(`Flow ${flowSlug} not found for ${team}`);
-
-  const metadata = {
-    flowSettings: flow.flowSettings,
-    flowAnalyticsLink: flow.flowAnalyticsLink,
-    isFlowPublished: flow.publishedFlowsAggregate?.aggregate.count > 0,
-  };
-  return metadata;
-};
+const SettingsContainer = () => (
+  <Box sx={{ width: "100%", p: 2, overflowY: "auto" }}>
+    <View />
+  </Box>
+)
 
 const routes = compose(
   withData((req) => ({
     flow: req.params.flow.split(",")[0],
   })),
 
-  withView(async (req) => {
-    const [flow, ...breadcrumbs] = req.params.flow.split(",");
-    const { flowSettings, flowAnalyticsLink, isFlowPublished } =
-      await getFlowMetadata(flow, req.params.team);
-    useStore.setState({ flowSettings, flowAnalyticsLink, isFlowPublished });
-
-    return (
-      <>
-        <ErrorBoundary FallbackComponent={ErrorFallback}>
-          <TestEnvironmentBanner />
-          <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
-        </ErrorBoundary>
-        <View />
-      </>
-    );
-  }),
+  withView(flowEditorView),
 
   mount({
     "/": route(async (req) => {
       return {
         title: makeTitle([req.params.team, req.params.flow].join("/")),
-        view: <span />,
-      };
+        view: () => {
+          const [flow, ...breadcrumbs] = req.params.flow.split(",");
+          return (
+            <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
+          )
+        }
+      }
     }),
 
-    "/nodes": nodeRoutes,
+    "/nodes": compose(
+      withView((req) => {
+        const [flow, ...breadcrumbs] = req.params.flow.split(",");
+        return (
+          <>
+            <FlowEditor key={flow} flow={flow} breadcrumbs={breadcrumbs} />
+            <View />
+          </>
+        );
+      }),
+      nodeRoutes,
+    ),
 
     "/settings": lazy(() => import("./flowSettings")),
+
+    "/service": compose(
+      withView(SettingsContainer),
+
+      route(async (req) => ({
+        title: makeTitle([req.params.team, req.params.flow, "service"].join("/")),
+        view: ServiceSettings
+      })),
+    ),
+
+    "/service-flags": compose(
+      withView(SettingsContainer),
+
+      route(async (req) => ({
+        title: makeTitle([req.params.team, req.params.flow, "service-flags"].join("/")),
+        view: ServiceFlags
+      })),
+
+    ),
+
+    "/data": compose(
+      withView(SettingsContainer),
+
+      route(async (req) => ({
+        title: makeTitle([req.params.team, req.params.flow, "data"].join("/")),
+        view: DataManagerSettings
+      })),
+    ),
+
+    "/submissions-log": compose(
```

Comment:
I've not hidden these routes behind a feature flag as there's nothing sensitive or different here, and they've not very discoverable.